src/main/java/frc/robot/commands/AccumulatorCommand.java

```diff
@@ -77,21 +79,26 @@ else if (RobotContainer.Xbox.getTriggerAxis(Hand.kLeft) > 0.05) {
       }
       else {
           end(true);   //idk what happens if u put false here
```

Comment:
remove this line.
have a motorPower variable. if motorPower != 0 and trigger == 0, then setPower(0,0,0)
when you are reading from trigger initially, update motorPower